src/notifications/common.ts

```diff
@@ -148,7 +148,7 @@ export const saveNotificationPreference = async (
       .createQueryBuilder()
       .insert()
       .values(params)
-      .orIgnore()
+      .orUpdate(['status'], ['referenceId', 'userId', 'notificationType'])
```

Comment:
@sshanzel I would like your eyes here specifically, so I noticed now that we have new status `subscribed` it's possible to (due to frontend being out of sync for any reason) to send subscribe when you are muted and vice versa. 

So I updated to instead of ignore always overwrite `status` on conflict. Nothing change functionality wise we only additionally support this case.

Reply:
Yeah, this makes sense. Currently, we remove the row when the user unsubscribe, so the expected behavior as of the moment is to always insert a new row.

Just not sure how the `orUpdate` works, but if it has been verified that it applies the correct condition of which entity to update and it sets the right values, we should be good to go 🚢

Reply:
Nice, in that case we can be sure that this is the only one that updates 🚀